### PR TITLE
[Submitit] Add test for example app

### PR DESCRIPTION
**Pull request by jrapin**
## Motivation

Making sure the example app works, and investigate Job environment initialization issue

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebookresearch/hydra/blob/master/CONTRIBUTING.md)?

Yes

## Test Plan

well, it's a test

## Related Issues and PRs

#849 

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: website/docs/plugins/submitit_launcher.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ params:
 
 You can set all these parameters in your configuration file and/or override them in the commandline: 
 ```text
-python foo.py hydra/launcher=submitit_slurm hydra.launcher.params.timeout_min=3
+python foo.py -m hydra/launcher=submitit_slurm hydra.launcher.params.timeout_min=3
```
Comment (jrapin) [inline website/docs/plugins/submitit_launcher.md:76]: is it needed? I'm a bit confused

Comment (omry) [reply website/docs/plugins/submitit_launcher.md:76]: Yes. see my comment on the original issue.

**Comment by omry** [comment]
See the part about isort in plugins [here](https://hydra.cc/docs/next/development/style_guide).

**Comment by jrapin** [comment]
> See the part about isort in plugins [here](https://hydra.cc/docs/next/development/style_guide).

`FIX=1 nox -s lint_plugins` did not fix anything after I removed all DS_store folders on my mac which made it crash...
so I suppose I should uninstall isort and reinstall it, and same on precommit but they dit not match anyway the last time. I begin to really dislike isort :) on this diff I spent more time trying to fix it that everything else. And that's not the first time

**Pull request merged**